lib/vmfloaty/ssh.rb

```diff
@@ -30,7 +30,12 @@ def self.ssh(verbose, host_os, token, url)
       end
 
       hostname = "#{response[host_os]["hostname"]}.#{response["domain"]}"
-      cmd = "#{ssh_path} #{user}@#{hostname}"
+      ssh_options= ["-oStrictHostKeyChecking=no", "-oUserKnownHostsFile=/dev/null"]
+      if ENV['VMFLOATY_SSH_KEY']
+        ssh_options << "-i #{ENV['VMFLOATY_SSH_KEY']}"
```

Comment:
No worries @underscorgan !!! 

@mckern  - That's a good idea. I haven't read any docs that suggest commander can do that, but if this tool does plan on supporting environment variables, I'd at least like a flag to go along with it so it's easy for users to discover in the help menu. 👍